run_tests/README.md

```diff
@@ -0,0 +1,49 @@
+# Automated Testing Script
+
+This folder contains a script that will run through the tests in this repo using an automated UI integration testing approach. Because this uses a real browser with real hardware, it must be run on a machine with bluetooth functionality and a bluetooth device within range.
+
+At this time, only the Chrome implementation of this script is complete. There are placeholders for future Firefox and IE implementations.
+
+## Exceptions
+
+This script is only intented to run through tests with a common UI pattern. This is:
+
+1. Open the test html page
+2. Click on the "Upload Device Code" button
+3. Operate the opened espruino page to complete code upload
+4. Press the "Run Test" button on the test html page
+
+The following tests are skipped because they require steps that the other tests do not.
+
+- https://webbluetoothcg.github.io/manual-tests/get_devices/ _(This test requires the user to manually revoke bluetooth permissions durin the test, and has a different button to finish the test.)_
```

Comment:
typo: durin -> during, on 1st and 3rd bullet points.

Reply:
```suggestion
- https://webbluetoothcg.github.io/manual-tests/get_devices/ _(This test requires the user to manually revoke bluetooth permissions during the test, and has a different button to finish the test.)_
```